.github/workflows/build.yml

```diff
@@ -63,6 +63,27 @@ jobs:
       id: hash-gcc
       run: echo "::set-output name=hashes::$(sha256sum Linux.flatc.binary.${{ matrix.cxx }}.zip | base64 -w0)"
 
+  build-linux-s390x:
+    name: Build Linux on S390X Arch
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: uraimo/run-on-arch-action@v2
+        name: Run commands
+        id: runcmd
+        with:
+          arch: s390x
+          distro: ubuntu_latest
+          install: |
+            apt-get update -q -y
+            apt-get -y install cmake
+            apt-get -y install make
+            apt-get -y install g++
+          run: |
+            cmake -G "Unix Makefiles" -DCMAKE_BUILD_TYPE=Release
+            make -j
+            ./flattests
```

Comment:
We should also look to make sure that it is running Big Endian. I wonder if you can add a print statement somewhere in the flattests that checks the ENDIAN flag and prints out a message of the endianess?

Reply:
I added a command to print out the byte order.

Reply:
I verified before making this on schedule, it is printing out big endian as expected. You can check it out here https://github.com/google/flatbuffers/actions/runs/3682731137/jobs/6230612636